<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Metronome</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1,0,1,0,0,0,0,0,0">
    <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="lblTempo">
      <property name="text">
       <string>Tempo</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="8" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QSlider" name="sldTempo">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLineEdit" name="edtTempo">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="lblBPM">
      <property name="text">
       <string>BPM</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
     <widget class="QLineEdit" name="edtBPM">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="text">
       <string>60</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="lblVolume">
      <property name="text">
       <string>Volume</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="8" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QSlider" name="sldVolume">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="2" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLineEdit" name="edtVolume">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="text">
       <string>0.35</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1" alignment="Qt::AlignmentFlag::AlignTop">
     <widget class="QPushButton" name="btnReset">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="lblPause">
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
     <widget class="QCheckBox" name="chkPause"/>
    </item>
    <item row="5" column="1" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignBottom">
     <widget class="QLabel" name="lblAccent1">
      <property name="text">
       <string>Accent</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="spnAccent1">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>128</number>
      </property>
     </widget>
    </item>
    <item row="7" column="1" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignBottom">
     <widget class="QLabel" name="lblAccent2">
      <property name="text">
       <string>Sub-Accent</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="spnAccent2">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>128</number>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="btnPreset1"/>
    </item>
    <item row="10" column="1">
     <widget class="QPushButton" name="btnPreset2"/>
    </item>
    <item row="10" column="2">
     <widget class="QPushButton" name="btnPreset3"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
